python/LICENSE.md

```diff
@@ -0,0 +1,7 @@
+Copyright 2017 Jörg Thalheim
```

Comment:
@Mic92 I copied the Pythonix license as-is, but I'm not sure if this still makes sense when it's upstreamed. Do you wish to keep the license as is or are you willing to change to the [Nix source license](https://github.com/NixOS/nix/blob/master/COPYING)

Reply:
@infinisil you can change the license. I had a smaller number of contributor that you might want to also ask.

Reply:
Thanks! Other than you there's only @adisbladis who only [added added a single word](https://github.com/Mic92/pythonix/commit/f540a8ffe4723bc59c7a1eb5f5abbff8c8a8450f), I think we can assume he'll be okay with this :)